### ISSUE TITLE
Not working with QT6

### ISSUE THREAD

**Issue by Erik262** [Shirajuki/anki-redesign#20]
Unfortunately, it does not work on the new qt6 with Anki 2.1.50

Getting this error while starting Anki:

`When loading '⁨Anki-redesign⁩':
⁨Traceback (most recent call last):
  File "aqt.addons", line 244, in loadAddons
  File "/Users/erik/Library/Application Support/Anki2/addons21/308574457/__init__.py", line 620, in <module>
    update_theme()
  File "/Users/erik/Library/Application Support/Anki2/addons21/308574457/__init__.py", line 557, in update_theme
    apply_theme(ncolors)
  File "/Users/erik/Library/Application Support/Anki2/addons21/308574457/__init__.py", line 564, in apply_theme
    QPalette.ColorRole.Background: "WINDOW_BG",
  File "enum", line 429, in __getattr__
AttributeError: Background`

**Comment by Shirajuki** [comment]
Hello hello!
First of all, thank you very much for the issue, Anki-redesign is as of now not compatible with 2.1.50 since I haven't really thoroughly begun implementing the fixes for the new update of QT6 yet as it seemed that there wasn't that much interest in that.

However, I'll see if I'll have the time to test things out and making the addon compatible with the new update, in the next few days so you can try Anki-redesign out too. :D

**Comment by Erik262** [comment]
Wow, that would be great! since Anki itself says that macOS users with M1 chips have to take the qt6 version for better performance. Actually, the addon kinda works somehow but looks really strange and ugly but it works 😅. At the very first glance, it does not look like you have to do so much work to fix this. But I will wait and try :) Thanks!

**Comment by nfwyst** [comment]
thank you very much

**Comment by Shirajuki** [comment]
Hello again! I have some very good news! It seems that the fix was very simple as I had used an old ColorRule (Background) on applying the theme palette swap which wasn't supported anymore. This was fairly easy to fix, and I have manually tested the add-on to be working on 2.1.50 correctly now 🥳🥳. I am however on a train home from easter break with no wifi available atm, so the update will be pushed tonight instead. 💪
